### ISSUE TITLE
Accessibility update for Accordion Module

### ISSUE THREAD

**Issue by Robanna777** [beaverbuilder/feature-requests#327]
**Is your feature request related to a problem? Please describe.**
Accessibility issue with the accordion module. axe-core 4.6/lighthouse 10.1.1 gives an error: "Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children. Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions. Learn more about roles and required children elements."

**Describe the solution you'd like**
Update the accordion module to conform to current accessibility guidelines.

**Describe alternatives you've considered**
I've tried to rewrite the code with an override without success.

**Comment by codente** [comment]
We have an internal ticket to look into this but Axe Developer Tools doesn't flag this and neither does WebAIM so there's more research needed.

**Comment by Robanna777** [comment]
Thank you. I'm happy this is getting looked at.

I replicated this issue on the demo site. Axe Developer Tools will give this error when Expand First item is set to Yes.

https://64551610e4cd2.demo.wpbeaverbuilder.com/demo/

**Comment by codente** [comment]
Thanks for the details - I see it now with that enabled in AxeDevTools but WAVE (WebAim) still doesn't flag it.  I've upated our internal ticket with these additional details.

**Comment by evokweb** [comment]
To add to this issue, I tested the accordion module with a screen reader, and I couldn't get the reader to recognize the content, only the label. Thanks for looking into this.

**Comment by Robanna777** [comment]
Lighthouse 12.0.0 now flags this as an issue.

**Comment by Robanna777** [comment]
Any update on this?

Colorado's state law requiring us to adhere to WCAG just went into effect, and this is the last error remaining on our site.

**Comment by gbissland** [comment]
We need an WCAG accessible accordion for an upcoming job. I know that is it going to be checked.
There is an add-on with what they say are 10 WCAG compliant modules. https://accessiblebb.com/
I haven't yet bought or tested it but I was going to try it.  

They look to be their own modules rather than building ontop the BB ones.

I'd love to know if these are 'simple' markup additions or improvements which could be added incorporated into the standard BB modules.
